### PR TITLE
renaming CNF Test Suite

### PR DESCRIPTION
**Pull request by lixuna**
refs #555

## Description
- Remove `conformance` from test suite name
- update urls for slack channels

## Issues:
Refs: #555

## How has this been tested:
 - [ ] Covered by existing integration testing
 - [ ] Added integration testing to cover
 - [ ] Verified all A/C passes
     * [ ] develop
     * [ ] master
     * [ ] tag/other branch
 - [ ] Test environment
    * [ ] Shared Packet K8s cluster
    * [ ] New Packet K8s cluster
    * [ ] Kind cluster
 - [x] Have not tested

## Types of changes:
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [x] Documentation update

## Checklist:
**Documentation**
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] No updates required.

**Code Review**
- [ ] Does the test handle fatal exceptions, ie. rescue block

**Issue**
- [ ] Tasks in issue are checked off

### CODE REVIEW

**Review by agentpoyo — APPROVED** [review]
LGTM 

And yes, now I noticed it's still a "Draft".  ;)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Before starting work on a major feature, please reach out to us via [GitHub Issu
 
 Issues
 ---
-Issues are used as the primary method for tracking items in the CNF Conformance initiative. Please self-assign an issue to yourself when you start to work on it so we don't duplicate work :) 
+Issues are used as the primary method for tracking items in the CNF CNF Test Suite initiative. Please self-assign an issue to yourself when you start to work on it so we don't duplicate work :)
```
Comment (lixuna) [inline CONTRIBUTING.md:21]:
```suggestion
Issues are used as the primary method for tracking items in the CNF Test Suite initiative. Please self-assign an issue to yourself when you start to work on it so we don't duplicate work :) 
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -53,14 +53,14 @@ crystal src/cnf-conformance.cr test_name verbose
...
-**3. New Conformance Tests:**
+**3. New CNF Test Suite Tests:**
 - To request a new workload test, please create a new issue using the [**New Workload Test**](https://github.com/cncf/cnf-conformance/issues/new?assignees=&labels=workload&template=new-workload-test.md&title=%5BWorkload%5D) Template
 - To request a new platform test, please create a new issue using the [**New Platform Test**](https://github.com/cncf/cnf-conformance/issues/new?assignees=&labels=platform&template=new-platform-test.md&title=%5BPlatform%5D) Template
 
 **4. New CNF Example:** 
 To suggest a new CNF, please create a GitHub issue using the [New Example CNF template](https://github.com/cncf/cnf-conformance/issues/new?assignees=&labels=example+CNF&template=new-example-cnf.md&title=%5BCNF%5D).
 
-To install the CNF Conformance test suite and run a CNF, follow instructions at:
+To install the CNF CNF Test Suite and run a CNF, follow instructions at:
```
Comment (lixuna) [inline CONTRIBUTING.md:63]:
```suggestion
To install the CNF Test Suite and run a CNF, follow instructions at:
```

File: FAQ.md [hunk 3]
```diff
@@ -1,43 +1,43 @@
...
+CNF Test Suite Frequently Asked Questions 
 ---
 
 ### General
-<details> <summary>What is CNF Conformance?</summary>
+<details> <summary>What is the CNF Test Suite?</summary>
 <p>
 
- - The CNF Conformance program enables interoperability of Cloud native Network Functions (CNFs) from multiple vendors running on top of Kubernetes. The goal is to provide an open source test suite to demonstrate conformance and implementation of best practices for both open and closed source Cloud native Network Functions.
+ - The CNF Conformance program enables interoperability of Cloud native Network Functions (CNFs) from multiple vendors running on top of Kubernetes. The CNF Test Suite's goal is to provide an open source test suite to demonstrate conformance and implementation of best practices for both open and closed source Cloud native Network Functions.
```
Comment (lixuna) [inline FAQ.md:8]:
```suggestion
 - The CNF Conformance program enables interoperability of Cloud native Network Functions (CNFs) from multiple vendors running on top of Kubernetes. The CNF Test Suite's goal is to provide an open source test suite to demonstrate conformance and implementation of best practices for both open and closed source Cloud Native Network Functions.
```

File: FAQ.md [hunk 4]
```diff
@@ -1,43 +1,43 @@
...
 
 </p>
 </details>
 
-<details> <summary>Does CNF Conformance have a slack channel?</summary>
+<details> <summary>Does CNF Test Suite have a slack channel?</summary>
 <p>
 
- - Yes. We have several two channels on [https://slack.cncf.io/](https://slack.cncf.io/), cnf-conformance and cnf-conformance-dev.
+ - Yes. We have several two channels on [https://slack.cncf.io/](https://slack.cncf.io/), cnf-testsuite and cnf-testsuite-dev.
```
Comment (lixuna) [inline FAQ.md:32]:
```suggestion
 - Yes. We have two channels on [https://slack.cncf.io/](https://slack.cncf.io/), cnf-testsuite and cnf-testsuite-dev.
```

File: FAQ.md [hunk 5]
```diff
@@ -110,12 +110,12 @@ CNF Conformance Test Suite Frequently Asked Questions
...
 <p>
 
- - All test results are stored in the results/ directory of where you installed the CNF Conformance suite in yaml format.
+ - All test results are stored in the results/ directory of where you installed the CNF Test Suite in yaml format.
 
 </p>
 </details>
 
-<details> <summary>Why is CNF Conformance written in crystal and not in other languages like Go?</summary>
+<details> <summary>Why is the CNF Test Suite written in crystal and not in other languages like Go?</summary>
```
Comment (lixuna) [inline FAQ.md:118]:
```suggestion
<details> <summary>Why is the CNF Test Suite written in Crystal and not in other languages like Go?</summary>
```

File: FAQ.md [hunk 6]
```diff
@@ -110,12 +110,12 @@ CNF Conformance Test Suite Frequently Asked Questions
...
 <p>
 
  - The short answer is Crystal fit the criteria we looked at in a language at the time which needed to run external programs/test suites and internal tests - [Taylor Carpenter](https://app.slack.com/client/T08PSQ7BQ/G019HM3K54H/user_profile/U7HCKCW90) via https://slack.cncf.io/
```
Comment (lixuna) [inline FAQ.md:121]:
```suggestion
 - The short answer is [Crystal](https://crystal-lang.org) fit the criteria we looked at in a language at the time which needed to run external programs/test suites and internal tests - [Taylor Carpenter](https://app.slack.com/client/T08PSQ7BQ/G019HM3K54H/user_profile/U7HCKCW90) via https://slack.cncf.io/ 
```

File: INSTALL.md [hunk 7]
```diff
@@ -14,7 +14,7 @@ This INSTALL guide will detail the minimum requirements needed for cnf-conforman
 This will detail the required minimum requirements needed in order to support cnf-conformance.
 
 #### Minimum Requirements
-* **kubernetes cluster** *(Working k8s cluster, see [supported k8s and installation details](#Details-on-supported-k8s-clusters-and-installation) on installation.*
+* **kubernetes cluster** *(Working K8s cluster, see [supported K8s and installation details](#Details-on-supported-k8s-clusters-and-installation) on installation.*
```
Comment (lixuna) [inline INSTALL.md:17]:
```suggestion
* **Kubernetes cluster** *(Working K8s cluster, see [supported K8s and installation details](#Details-on-supported-k8s-clusters-and-installation) on installation.*
```

File: INSTALL.md [hunk 8]
```diff
@@ -14,7 +14,7 @@ This INSTALL guide will detail the minimum requirements needed for cnf-conforman
...
 * **kubectl** *(run commands against k8 clusters, see [installing kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) for more details.*
```
Comment (lixuna) [inline INSTALL.md:18]:
```suggestion
* **kubectl** *(run commands against K8s clusters, see [installing kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) for more details.*
```

File: README.md [hunk 9]
```diff
@@ -1,11 +1,11 @@
-# CNF Conformance Test Suite
+# CNF Test Suite
 | Main |
 |---|
 |[![Build Status](https://github.com/cncf/cnf-conformance/workflows/Crystal%20Specs/badge.svg)](https://github.com/cncf/cnf-conformance/actions)|
 
-The CNF Conformance test suite is a tool that makes it possible to validate telco applications, aka Cloud native Network Functions (CNFs), and the underlying Telecom platforms adherence to Cloud native principles and best practices.
+The CNF Test Suite is a tool that makes it possible to validate telco applications, aka Cloud native Network Functions (CNFs), and the underlying Telecom platforms adherence to Cloud native principles and best practices.
```
Comment (lixuna) [inline README.md:6]:
```suggestion
The CNF Test Suite is a tool that makes it possible to validate telco applications, aka Cloud Native Network Functions (CNFs), and the underlying Telecom platforms adherence to Cloud native principles and best practices.
```

Comment (lixuna) [inline README.md:6]:
```suggestion
The CNF Test Suite is a tool that makes it possible to validate telco applications, aka Cloud Native Network Functions (CNFs), and the underlying Telecom platforms adherence to Cloud Native principles and best practices.
```

**Comment by lixuna** [comment]
@agentpoyo I made a few changes after your initial review

**Review by taylor — APPROVED** [review]
wow. a bunch of changes!  lgtm.  thx @lixuna

**Pull request merged**